tools/c7n_azure/c7n_azure/resources/arm.py

```diff
@@ -71,6 +72,7 @@ def register_arm_specific(registry, _):
                 klass.action_registry.register('untag', RemoveTag)
                 klass.action_registry.register('auto-tag-user', AutoTagUser)
                 klass.action_registry.register('tag-trim', TagTrim)
+                klass.action_registry.register('logic-app', LogicAppAction)
```

Comment:
We should move it to `QueryResourceManager`, it is not arm-specific action

Reply:
Yeah for sure!